addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -418,8 +417,11 @@ export class LinkPlugin extends Plugin {
             };
 
             if (linkEl.isConnected) {
+                const imageNode = this.linkElement.querySelector("img");
```

Comment:
I'm a bit confused here, in the top condition you skip if there is an image, but here you check if there is an image, can you clarify how this works and in what cases, and if needed add a comment about it

Reply:
@Mtaylorr good point, we dont need it actually. thanks for the feedback.